Description: Remove UTF-8 warning
 Even though upstream must be right, this warning is breaking other software
 that are parsing the output, for example the rabbitmq puppet module.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/912207
Forwarded: no
Last-Update: 2018-10-30

Index: elixir-lang/lib/elixir/src/elixir.erl
===================================================================
--- elixir-lang.orig/lib/elixir/src/elixir.erl
+++ elixir-lang/lib/elixir/src/elixir.erl
@@ -137,11 +137,6 @@ check_endianness() ->
 
 check_file_encoding(Encoding) ->
   case Encoding of
-    latin1 ->
-      io:format(standard_error,
-        "warning: the VM is running with native name encoding of latin1 which may cause "
-        "Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 "
-        "(which can be verified by running \"locale\" in your shell)~n", []);
     _ ->
       ok
   end.
